### ISSUE TITLE
Standard-Dialect.xml links to tutorials/2.1 instead of 3.0

### ISSUE THREAD

**Issue by maxxyme** [thymeleaf/thymeleaf#557]
In this file: https://github.com/thymeleaf/thymeleaf/blob/3.0-master/src/main/resources/org/thymeleaf/xml/thymeleaf/Standard-Dialect.xml

the links found point to http://www.thymeleaf.org/doc/tutorials/2.1/usingthymeleaf.html instead of http://www.thymeleaf.org/doc/tutorials/3.0/usingthymeleaf.html

**Comment by ultraq** [comment]
I've updated these already, but they live in the `3.0-dev` branch which hasn't been merged into master just yet.  (https://github.com/thymeleaf/thymeleaf/blob/3.0-dev/src/main/resources/org/thymeleaf/xml/thymeleaf/Standard-Dialect.xml)  They should be merged before the next release.

**Comment by maxxyme** [comment]
OK, I should've check the dev branch too. Sorry for that.

**Comment by ultraq** [comment]
That's OK, not many people know to look there 🙂
